src/engines/kotor/gui/options/gameplay.cpp

```diff
@@ -63,6 +63,21 @@ void OptionsGameplayMenu::show() {
 
 	_difficulty = CLIP(ConfigMan.getInt("Difficulty Level", 0), 0, 2);
 	updateDifficulty(_difficulty);
+
+	_autoLevelUp = ConfigMan.getBool("Auto Level Up", false);
```

Comment:
Hmm, please don't spaces in the config options. They basically map 1:1 to command line switches, and there a space obviously won't work. Also, while they are case-insensitive, I prefer if xoreos itself writes all-lower-case.

See how NWN does it:
`ConfigMan.setBool("mouseoverfeedback", mouseoverFeedback, true);`

Reply:
I was trying to match the "Difficulty Level" that was already there but I'll change them.